zipline/pipeline/loaders/utils.py

```diff
@@ -237,11 +237,15 @@ def normalize_timestamp_to_query_time(df,
         _midnight,
         include_end=False,
     )
-    # for all of the times that are greater than our query time add 1
-    # day and truncate to the date
+    # For all of the times that are greater than our query time add 1
+    # day and truncate to the date.
+    # We normalize twice here because of a bug in pandas 16.0.1 that causes
```

Comment:
The pandas version is 0.16.1, not 16.0.1.  Is there a link to the change that we can add here so that we know when we can remove this?